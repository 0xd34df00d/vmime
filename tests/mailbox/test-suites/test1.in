My (this is a comment)name <me(another \)comment) @    	somewhere(else).com>
