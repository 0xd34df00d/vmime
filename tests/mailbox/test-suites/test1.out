MAILBOX[name=[[us-ascii,My name]],email=me@somewhere.com]
